examples/fingerprint_template_file_compare.py

```diff
@@ -0,0 +1,174 @@
+import serial
+import adafruit_fingerprint
+
+
+# import board
+# uart = busio.UART(board.TX, board.RX, baudrate=57600)
+
+# If using with a computer such as Linux/RaspberryPi, Mac, Windows with USB/serial converter:
+uart = serial.Serial("COM4", baudrate=57600, timeout=1)
+
+# If using with Linux/Raspberry Pi and hardware UART:
+# uart = serial.Serial("/dev/ttyS0", baudrate=57600, timeout=1)
+
+# If using with Linux/Raspberry Pi 3 with pi3-disable-bte
+# uart = serial.Serial("/dev/ttyAMA0", baudrate=57600, timeout=1)
+
+finger = adafruit_fingerprint.Adafruit_Fingerprint(uart)
+
+##################################################
+
+
+def sensor_reset():
+    """Reset sensor"""
+    print("Resetting sensor...")
+    if finger.soft_reset() != adafruit_fingerprint.OK:
+        print("Unable to reset sensor!")
+    print("Sensor is reset.")
+
+
+# pylint: disable=too-many-branches
+def fingerprint_check_file():
+    """Compares a new fingerprint template to an existing template stored in a file
+       This is useful when templates are stored centrally (i.e. in a database)"""
+    print("Waiting for finger print...")
+    finger.set_led(color=3, mode=1)
```

Comment:
The LED control is specifically to the R503 sensor -- should this example be limited to the R503
Will it work on other supported sensors?

Reply:
Good point. Not sure about this. We can designate the example as R503.

Reply:
Hopefully another reviewer has one and can test it. In any case, we may need an r503 version for the LED, so I'd suggest just changing the name of this example to reflect that.